CI.md

```diff
@@ -125,6 +125,33 @@ Service Account shall have following permissions ([IAM roles](https://cloud.goog
 | Java Wordcount Direct Runner | Runs Java WordCount example with Direct Runner.                                               | Yes              | Yes                   | Yes           | -                        |
 | Java Wordcount Dataflow      | Runs Java WordCount example with DataFlow Runner.                                             | -                | Yes                   | Yes           | Yes                      |
 
+### PreCommit Workflows
+
+#### Java MQTT IO - [job-precommit-java-mqtt-io.yml](.github/workflows/job-precommit-java-mqtt-io.yml)
```

Comment:
Could we remove section headers and make this a single table?

Reply:
Done!